app/services/kredits.js

```diff
@@ -136,6 +136,10 @@ export default Service.extend({
     });
   },
 
+  getKreditsDeployedNetwork() {
+    return Kredits.availableNetworks();
```

Comment:
I don't understand how one can get a single network from a list of networks without filtering here.

Reply:
mm I thought that there will be only the network where kredits is deployed at (etiher rinkeby, ropsten, mainnet...), or no ?

Reply:
We run a local network for development, making it at least two. But also, there could be more networks in the future, hence the plural in `availableNetworks()` (as opposed to `getNetwork()` or similar).

We talked about this in the call yesterday and @bumi had some good ideas. Would be good to write them down in this PR.

Reply:
yeah really sorry for missing the last calls... was trying to join but I miss for some reasons... I will be available on the next ones.